cmd/helm/get_dependencies.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"io"
+
+	"github.com/gosuri/uitable"
+	"github.com/spf13/cobra"
+
+	"helm.sh/helm/v3/cmd/helm/require"
+	"helm.sh/helm/v3/internal/completion"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/cli/output"
+)
+
+var getDependenciesHelp = `
+This command list all dependencies for a given release.
+`
+
+type dependencyElement struct {
+	Name       string `json:"name"`
+	Version    string `json:"version,omitempty"`
+	Repository string `json:"repository"`
```

Comment:
`Alias` might be worth adding: https://github.com/helm/helm/blob/master/pkg/chart/dependency.go#L49